lib/src/intl_suggestors/intl_migrator.dart

```diff
@@ -81,6 +81,9 @@ class IntlMigrator extends ComponentUsageMigrator {
 
   IntlMigrator(this._className, this._messages);
 
+  String get ignoreStatement => 'IGNORE STATEMENT - INTL';
```

Comment:
```suggestion
  String get ignoreStatement => 'ignore: intl';
```

Reply:
I can change this, but I was trying to follow the pattern set by `// FIXME - INTL`. Should I still change it?

Reply:
Ignoring other things in dart follows this pattern (lints for example) so I think it would be the default format that devs would go to. I like the `// ignore:` prefix but could be convinced to provide a more specific word than just `intl` .... maybe `intl_codemod` or something.

Reply:
See below